### DISCUSSION TITLE
Pre-loading value in cascaded drop down within an edit form

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#11023] [Help]
Hi,

I am struggling to work out how to set the default value of a cascaded drop down when loading an existing item.

**Scenario** 
We are building a tool to report on projects, a project can have an agency and each agency has its own portfolios.

When registering a new project, the new item form pops up and a user can select the agency and a related portfolio of that agency. This is successfully saving to MSSQL.

To facilitate the cascade the agency is a relationship picker and portfolio is an options picker that is within its own data provider with a filter based on the selected agency.

**Issue**
When we go to amend a saved entry using the Update form type, i can load the agency value but cannot work out how to load the saved value for portfolio as the options picker is within a data provider. 

I also need to allow the portfolio value to be changed based on the cascade. I can change portfolio to a relationship field and it will display the saved value, but i lose the cascade functionality on the change form.

**SQL**
We have the following SQL tables to faciliatate this created within Budibase:

- projects
- client_agency
- porfolio

With the following relationships defined within Budibase:

- client_agency -> project (1:M)
- client_agency -> portfolio (1:M)

**Comment by melohagan** [comment]
Hey @benaunt 

I may not be entirely visualising your use-case, but my instinct is that this feature needs to be implemented to achieve what you are looking to do: https://github.com/Budibase/budibase/issues/7810

**Reply by benaunt** [reply]
If you have a create form that has cascading drop downs, you can successfully perform your cascades and save to a specified data source.

If you then want to load that data in an update form say to amend one of the cascaded selections. You would want the form to:

- Maintain all previously selected cascaded drop downs so that if you click a child drop down its already filtered
- Display the previously saved values for those drop downs

Is there any way to achieve that?

Thanks
